packages/Filters/stories/MockServerFilter/api.js

```diff
@@ -0,0 +1,17 @@
+import data from "../ShowcaseApp/data";
+
+export const fetchAll = () => {
+  return new Promise(resolve => {
+    setTimeout(() => {
+      resolve(data);
+    }, 2000);
+  });
+};
+
+export const filterBy = () => {
+  return new Promise(resolve => {
+    setTimeout(() => {
+      resolve([data[1]]);
+    }, 2000);
+  });
+};
```

Comment:
LOL
```suggestion
import data from "../ShowcaseApp/data";

export const fetchAll = () => {
  return new Promise(resolve => {
    setTimeout(() => {
      resolve(data);
    }, 500);
  });
};

export const filterBy = () => {
  return new Promise(resolve => {
    setTimeout(() => {
      resolve([data[1]]);
    }, 500);
  });
};
```

Reply:
I'd prefer 800 to have the experience of the spinner 😅